templates/sub-01/ses-01/eeg/sub-01_task-FullExample_acq-CTF_run-1_proc-sss_eeg.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
@CPernet, what is the purpose of this file? Does it show something that has not been shown with the other two files?